### DISCUSSION TITLE
Pipelined Container taking restart while deploy CWAG

### DISCUSSION THREAD

**Discussion by Aniket621** [magma/magma#15287] [Communications]
Hi all, 

I am trying to deploy CWAG but i am seeing that pipelined container is restart mode.

![image](https://github.com/magma/magma/assets/104416203/77e6926d-2781-4c99-a0bf-c5bdfcc55f3a)

Container Logs 
[pipelined_logs.txt](https://github.com/magma/magma/files/12377914/pipelined_logs.txt)

**Comment by Aniket621** [comment]
Hi all anyone idea regarding the above issue

**Comment by Aniket621** [comment]
Hi all this above issue is resolved.

**Reply by panyogesh** [reply]
Thanks
